linode/manager
# refactor: [M3-6802] - Remove Linodes Redux store

## Description 📝
Completes the RQ Linodes migration by removing the Linodes Redux store and refactoring the last few places that still used it.

## Major Changes 🔄
- Deleted Linode Redux store (`src/store/linodes`)
- Migrated `BackupDrawer` and `SelectPlanPanel` to RQ Linode hooks/HOCs
- Moved Linodes utilities to new `src/utilities/linodes.ts` file
- Slightly refactored `ConfigSelect` to display no options and error messages

## How to test 🧪
1. Ensure all actions on Linodes (i.e., create, rename, reboot, delete, migrate, etc.) continue to work as expected 
2. Ensure enabling/disabling backups (for single Linode, for all Linodes, auto-enroll) all continue to work as expected
3. Ensure `SelectPlanPanel` continues to work as expected
4. Ensure all other components in the App that fetch Linodes continue to work as expected, e.g.:
    - Setting up NodeBalancers for Linodes
    - Attaching volumes to Linodes
    - Adding Linodes to Firewalls
    - Creating Kubernetes clusters
5. Ensure unit tests and E2E test pass
